tests/drivers/watchdog/wdt_basic_reset_none/src/main.c

```diff
@@ -20,10 +20,18 @@
 #endif
 
 #define WDT_FEED_TRIES		2
-#define WDT_MAX_WINDOW		1000
-#define WDT_TIMEOUT		K_MSEC(1100)
-#define SLEEP_TIME		K_MSEC(500)
 #define WDT_TEST_CB_TEST_VALUE	0xCB
+#define WDT_TIMEOUT_VALUE	CONFIG_TEST_WDT_MAX_WINDOW_TIME + 10
+
+#if defined(CONFIG_WDT_NXP_EWM)
+#define WDT_SETUP_FLAGS		0
+#define WDT_TIMEOUT		K_TICKS(WDT_TIMEOUT_VALUE)
```

Comment:
Why does the `NXP_EWM` get the value on ticks?

Reply:
The watchdog window to feed the watchdog is in the Microsecond range. The Tick value is the most accurate speed to test this EWM without having to worry about the test unintentionally missing a watchdog feed.